test/utils/BaseTest.sol

```diff
@@ -25,21 +25,27 @@ contract BaseSetup is Test {
     dev = users[1];
     alice = users[2];
     bob = users[3];
-    carole = users[4];
+    charlie = users[4];
     dave = users[5];
     vm.label(owner, "Owner");
     vm.label(dev, "Developer");
     vm.label(alice, "Alice");
     vm.label(bob, "Bob");
-    vm.label(carole, "Carole");
+    vm.label(charlie, "Charlie");
     vm.label(dave, "Dave");
   }
 
   function logObservations(TwabLib.Account memory account, uint256 amount) internal view {
+    console.log("-- Observations --");
     for (uint256 i = 0; i < amount; i++) {
       ObservationLib.Observation memory observation = account.observations[i];
-      console.log("Observation: ", i, observation.cumulativeBalance, observation.timestamp);
+      console.log(i, observation.cumulativeBalance, observation.balance, observation.timestamp);
     }
     console.log("--");
   }
+
+  function assertAlmostEqual(uint256 a, uint256 b, uint256 maxDelta) internal {
```

Comment:
I don't think this function is needed.
You can use [assertApproxEqAbs](https://book.getfoundry.sh/reference/forge-std/assertApproxEqAbs?highlight=assertApprox#assertapproxeqabs) if you want to test values with a delta.

Reply:
🤦 I was looking through the wrong section in the docs for the approximate checks.. <https://book.getfoundry.sh/reference/ds-test>